proposals/OBv3p0/respec-support/specification-wrapper.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!-- Markdown template for an extension document.
+        If you change the filename of the.md file, you need to
+        change the name in the src attribute of the
+        <script class="transclude> element below.
+    -->
+
+    <meta charset='utf-8'>
+    <script src='https://purl.imsglobal.org/spec/respec-ims-default.js' async class='remove'></script>
+    <script src='local-biblio.js' class='remove'></script>
+    <script src='contributors.js' class='remove'></script>
+    <script class='remove'>
+      var respecConfig = {
+        mainSpecTitle: "Open Badges",
+        mainSpecBiblioKey: "OB-20",
+        specTitle: "Open Badges 3.0",
+        shortName: "OBv3p0",
+        specNature: "normative", // spec nature is "normative" or "informative"
+        specType: "proposal", // spec type is "main", "cert", "impl", "errata" or other agreed upon string
+        specVersion: "3.0",
+        specStatus: "IMS Base Document", // specStatus: IMS Base Document, IMS Candidate Final (Public) or IMS Final Release
```

Comment:
Can you change `specStatus` to "Proposal" for now, we don't have a status for this yet so it won't look as nice, but Base Document isn't appropriate for proposals like this if PDFs and documents will be generated. 

We'll make the default template reflect this too so future proposals are already marked appropriately.

Reply:
Will do.

Reply:
All set

Reply:
Thanks!